gdcdictionary/schemas/clinical_trial.yaml

```diff
@@ -0,0 +1,342 @@
+
+id: "clinical_trial"
+title: Clinical Trial
+type: object
+namespace: https://niaid-clinical-trials-dictionary.org
+category: data_file
+program: '*'
+project: '*'
+description: >
+  The collection of all data related to a clinical trial.
+additionalProperties: false
+submittable: true
+validators: null
+
+systemProperties:
+  - id
+  - project_id
+  - created_datetime
+  - updated_datetime
+  - state
+
+links:
+  - exclusive: false
+    required: true
+    subgroup:
+    - name: core_metadata_collections
+      backref: clinical_trials
+      label: data_from
+      target_type: core_metadata_collection
+      multiplicity: many_to_one
+      required: false
+
+required:
+  - type
+  - submitter_id
+  - file_name
+  - file_size
+  - data_format
+  - md5sum
+  - data_category
+  - data_type
+
+uniqueKeys:
+  - [id]
+  - [project_id, submitter_id]
+
+# Case properties
+properties:
+
+  $ref: "_definitions.yaml#/data_file_properties"
+
+  type:
+    enum: [ "clinical_trial" ]
+
+  data_category:
+    term:
+      $ref: "_terms.yaml#/data_category"
+    type: string
+
+  data_type:
+    term:
+      $ref: "_terms.yaml#/data_type"
+    type: string
+
+  data_format:
+    term:
+      $ref: "_terms.yaml#/data_format"
+    type: string
+
+  submitted_date:
+    description: >
+      First Submitted Date
+    type: string
+
+  first_posted_date:
+    description: >
+      First Posted Date
+    type: string
+
+  last_update_posted_date:
+    description: >
+      Last Update Posted Date
+    type: string
+
+  actual_study_start_date:
+    description: >
+      Actual Study Start Date
+    type: string
+
+  actual_primary_completion_date:
+    description: >
+      Actual Primary Completion Date
+    type: string
+
+  title:
+    description: >
+      Name for this study (Brief Title)
+    type: string
+
+  official_title:
+    description: >
+      Official name for this study (Official Title)
+    type: string
+
+  description:
+    description: >
+      What this is study doing (Brief Summary)
+    type: string
+
+  detailed_description:
+    description: >
+      Detailed description of what this study is doing
+    type: string
+
+  category:
+    description: >
+      Classifications for the study (Study Type)
+    enum:
+      - "Investigational"
+      - "Observational"
+
+  phase:
+    description: >
+      Study Phase
+    enum:
+      - "n-a"
+      - "early-phase-1"
+      - "phase-1"
+      - "phase-1-phase-2"
+      - "phase-2"
+      - "phase-2-phase-3"
+      - "phase-3"
+      - "phase-4"
+  
+  study_design_allocation:
+    description: >
+      Study Design: Allocation
+    type: string
+
+  study_design_intervention:
+    description: >
+      Study Design: Intervention Model
+    type: string
+
+  study_design_masking:
+    description: >
+      Study Design: Masking
+    type: string
+
+  study_design_primary_purpose:
+    description: >
+      Study Design: Primary Purpose
+    type: string
```

Comment:
Same situation. Remove the colons from the description. We should probably re-craft the descriptions as well to read something like "The primary purpose of the study", "The intervention model of the study" etc.